exploration/zonal-means.ipynb

```diff
@@ -121,6 +121,63 @@
     "    zonal_means_to_csv(lst_zonal_means)\n",
     "    display(lst_zonal_means)"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "merged_precip_df = pd.DataFrame()\n",
+    "merged_precip_df[\"precip_m\"] = precip_zonal_means\n",
+    "merged_precip_df[\"precip_m_1\"] = precip_zonal_means.shift(1)\n",
+    "merged_precip_df[\"precip_m_2\"] = precip_zonal_means.shift(2)\n",
+    "merged_precip_df[\"precip_m_3\"] = precip_zonal_means.shift(3)\n",
+    "merged_precip_df"
```

Comment:
This doesn't quite do the trick. Take a close look at the first few rows of this DataFrame:

```plain
                                 precip_m  precip_m_1  precip_m_2  precip_m_3
year month location_period_id                                                
2009 10    92                  288.192780         NaN         NaN         NaN
           1162                 96.912056  288.192780         NaN         NaN
           1343                 69.695549   96.912056  288.192780         NaN
           1355                105.352257   69.695549   96.912056  288.192780
           1842                239.344131  105.352257   69.695549   96.912056
```

Notice the problem? (Hint: it's _not_ the `NaN` values -- simply look at the first value and how it's shifted values align with `location_period_id` [and potentially with year/month for other values])

Reply:
Ahhh I see it. Let me fix that 👀

Reply:
https://github.com/developmentseed/geospatial-ds-cholera-lab/pull/24/commits/323074e6f0bc9ade9221d5b58751712acf9d6b07 I updated it to group by the `location_period_id` and I believe this fixes the issue. @chuckwondo, do you mind reviewing again?